pkg/mfa/client.go

```diff
@@ -58,22 +58,32 @@ func (c *Client) init() {
 	}
 }
 
-// GetEnrollsOpts contains the options to create an Authentication Factor.
-type GetEnrollOpts struct {
+// Type represents the type of Authentication Factor
+type Type string
+
+// Constants that enumerate the available Types.
+const (
+	SMS  Type = "sms"
+	TOTP Type = "totp"
+)
```

Comment:
We already go these:
- `sso.ConnectionType`
- `directorysync.DirectoryType`
- `passwordless.PasswordlessSessionType`

Shall we explicitly call this `FactorType` or `AuthenticationFactorType`? (See next comment for the thoughts on the prefix).

Reply:
Updating to `FactorType`